testsuite/features/support/lavanda.rb

```diff
@@ -44,21 +44,11 @@ def full_hostname
     @in_full_hostname
   end
 
-  def private_ip
-    raise 'empty private_ip, something wrong' if @in_private_ip.empty?
-    @in_private_ip
-  end
-
   def public_ip
     raise 'empty public_ip, something wrong' if @in_public_ip.empty?
     @in_public_ip
   end
 
-  def private_interface
-    raise 'empty private_interface, something wrong' if @in_private_interface.empty?
-    @in_private_interface
-  end
-
```

Comment:
Keep it please. Let's only remove unused step definitions, but let's keep what we offer as part of our modules.